src/vs/workbench/contrib/positronAssistant/browser/languageModelModalDialog.tsx

```diff
@@ -280,6 +287,26 @@ const LanguageModelConfiguration = (props: React.PropsWithChildren<LanguageModel
 			});
 	}
 
+	const shouldCloseModal = async () => {
+		if (authMethod === AuthMethod.OAUTH && !source.signedIn && showProgress) {
+			return await props.positronModalDialogsService.showSimpleModalDialogPrompt(
+				localize('positron.newConnectionModalDialog.oauthInProgressTitle', "{0} Authentication in Progress", source.provider.displayName),
```

Comment:
I plan to replace instances of `newConnectionModalDialog` (this seems to be for the Connections Pane) with `modelProviderModalDialog`, but will do this after a review as it will cause 30+ lines to be added to the diff!

```suggestion
				localize('positron.modelProviderModalDialog.oauthInProgressTitle', "{0} Authentication in Progress", source.provider.displayName),
```

Reply:
updated in [rename newConnectionModalDialog --> languageModelProviderModalDialog](https://github.com/posit-dev/positron/pull/8029/commits/ddd54a6731d8dd7354b43747d6d9adab8e894c41)